content/blog/fs-advent-2021-fantomas-daemon/index.md

```diff
@@ -0,0 +1,100 @@
+This daemon process will act as an [LSP server](https://microsoft.github.io/language-server-protocol/) that send json messages back and forward to the `FantomasService`.
+As user of an editor, you don't need to worry about `Fantomas.Client`, just know that the editor will use a locally or globally installed version of `fantomas-tool`.
+Meaning, you control the version and you upgrade at your own pace.
+
+At the time of writing, there is support for VSCode (as part of the Ionide plugin).
+If you have Ionide installed, it will prompt you to install `fantomas-tool` if no compatible version can be found.
+After installing, you should be good to go as demonstrated in this video:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/1jb6K_HZaEY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Post-Daemon mode
+
+My hopes are that we can ship bug fixes to Fantomas at a high pace. If it gets on NuGet, it can be used in your IDE.
+Ever since 4.5, I have been releasing revisions on a frequent pace and people seem to have faith in this.
+Download numbers are significantly higher than say the first alpha of 4.4. Alpha and beta just don't have a stable ring to it for most people.
+I completely understand this, and that is why I'd like to move forward to a model were bugfixes are considered stable and people should not fear to upgrade.
+There is truth to this as these revisions mostly contain a fix for problem and nothing more.
+
+## What about Rider and Visual Studio?
+
+JetBrains has slightly different plans, although it is on their radar to support custom versions of Fantomas.
+They won't go with `Fantomas.Client` for technical reason as can be read in [this GitHub issue](https://github.com/JetBrains/resharper-fsharp/issues/320).
+
+Integration of `Fantomas.Client` is ongoing for the Visual Studio extension.
+Progress can be tracked [here](https://github.com/fsprojects/fantomas-for-vs/issues/21). Please don't hesitate to contribute any of the remaining work.
+
+## Acknowledgements
+
+During the development of `Fantomas.Client`, I had a lot of mentoring by [Chet Husk](https://github.com/baronfel) (the one and only `@baronfel` on GitHub).
+We working together on the Ionide integration in FSAutocomplete (or F-Sack as the cool kids say it).
+And over a timespan of several months we were able to come up with a prototype.
+
+For the Visual Studio integration I'd like to thanks [Christopher-Marcel Esser](https://github.com/bddckr) and [Asti](https://github.com/deviousasti).
+Christopher-Marcel implemented the port to `Fantomas.Client` and Asti is the core maintainer of the project.
+
+## Closing thoughts
+
+The whole Fantomas daemon thing is pretty mind-blowing. We can upgrade to different compiler version without a breeze.
```

Comment:
"We can upgrade to different compiler versions in a breeze."
(doing something easily is doing it 'in a breeze' or 'breezing by' because breezes are light and effortless)